src/demo-app/datepicker/datepicker-demo.ts

```diff
@@ -32,10 +34,24 @@ export class DatepickerDemo {
   lastDateChange: Date | null;
 
   dateFilter =
-      (date: Date) => !(date.getFullYear() % 2) && (date.getMonth() % 2) && !(date.getDate() % 2)
+    (date: Date) => !(date.getFullYear() % 2) && (date.getMonth() % 2) && !(date.getDate() % 2)
```

Comment:
nit: continuation lines indented 2

Reply:
done in 0faefc46a68fb3a15f91945858faad7a399c8c8a